lib/Targets/CMakeLists.txt

```diff
@@ -19,6 +21,28 @@ add_mlir_library(AIETargets
   ADFGenerateCppGraph.cpp
   AIEFlowsToJSON.cpp
   AIELLVMLink.cpp
+)
+
+if(AIE_ENABLE_GENERATE_CDODirect)
+  list(APPEND _sources AIETargetCDODirect.cpp)
+  add_compile_definitions(AIE_ENABLE_GENERATE_CDODirect)
+
+  include("${AIE_SOURCE_DIR}/runtime_lib/xaiengine/aiert.cmake")
+  set(XAIE_SOURCE "${AIE_SOURCE_DIR}/runtime_lib/xaiengine/aie-rt/driver/src")
+  add_aiert_headers(xaienginecdo
+    ${XAIE_SOURCE}
+    ${AIE_BINARY_DIR}/include
+    ${CMAKE_INSTALL_PREFIX}/include/xaiengine)
+  add_aiert_library(xaienginecdo ${XAIE_SOURCE} STATIC)
+  target_compile_options(xaienginecdo PRIVATE "-Wimplicit-function-declaration")
+  target_compile_definitions(xaienginecdo PRIVATE -D__AIECDO__)
+  install(TARGETS xaienginecdo DESTINATION ${CMAKE_INSTALL_PREFIX}/lib EXPORT xaienginecdo)
+  # just to make cmake happy since AIETargets will need to re-export
+  install(EXPORT xaienginecdo DESTINATION ${CMAKE_INSTALL_PREFIX}/cmake)
```

Comment:
Why compile this again?

Reply:
Compile it as a static archive so that it's statically linked in.

Reply:
If you want a static library, then we can probably just change the version that's already compiled to be a static library.  regardless, I don't think it makes sense to compile this in a completely different directory than where the source is.

Reply:
Sure I agree but the way the build is setup right now, with `ExternalProject`, you don't get a cmake target to link against.